packages/core/src/types.ts

```diff
@@ -1166,7 +1172,7 @@ export namespace SCXML {
 
 // Taken from RxJS
 export interface Unsubscribable {
-  unsubscribe(): any | void;
+  unsubscribe(): void;
```

Comment:
Hmm, I have `any` in here to prevent obscure type errors when `unsubscribe` returns something accidentally, e.g.,

```js
unsubscribe = () => something.off(handler); // might return something
```

As a way of signifying that whatever value is returned should be ignored. WDYT?

Reply:
Gonna experiment with different things in a moment and report back - my guess is that `unknown` is best for such cases.